src/main/java/org/esi/grable/menus/service/MenuService.java

```diff
@@ -0,0 +1,47 @@
+package org.esi.grable.menus.service;
+
+import org.esi.grable.menus.model.Menu;
+import org.esi.grable.menus.repository.MenuRepository;
+import org.esi.grable.restaurants.model.Restaurant;
+import org.esi.grable.restaurants.repository.RestaurantRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class MenuService {
+
+    @Autowired
+    private MenuRepository menuRepository;
+
+    @Autowired
+    private RestaurantRepository restaurantRepository;
```

Comment:
Ideally, I think we shouldn't use other repositories like that. But instead, use web client to fetch it. Example in [here](https://github.com/M-Gharib/ESI-W4/blob/main/product-service/src/main/java/com/esi/productservice/products/service/ProductService.java#L58)